.github/ISSUE_TEMPLATE/setup.yml

```diff
@@ -15,6 +15,7 @@ those tickets.
 
 - [ ] Create a README.
 - [ ] Set up [Tugboat](https://dashboard.tugboat.qa/).
+- [ ] Create an AWS user and S3 bucket with an access policy and bucket lifecycle rule for database backups.
```

Comment:
We need to link this to: https://github.com/ChromaticHQ/Chromatic-Handbook/blob/main/technical/database-backup-aws-s3.md

Thoughts on what text we should link, or should I add a second sentence?